plugins/lime-plugin-pirania/src/components/createVoucherForm.js

```diff
@@ -2,9 +2,11 @@ import { h } from "preact";
 import { useState } from "preact/hooks";
 import I18n from "i18n-js";
 import style from "../style.less";
+import Loading from "../../../../src/components/loading";
 const maxLength = 100;
 
 const CreateVoucherForm = ({ submitVoucher }) => {
+	const [loading, setLoading] = useState(false);
```

Comment:
ReactQuery's mutations give us handy variables to track the query status like isLoading. Which you can use instead of setting up a useState hook by your own.

Example:
```
const [addVoucher, {isLoading}] = useAddVoucher()
````